expressEngine/product/service/product.js

```diff
@@ -0,0 +1,40 @@
+// import service libraries
+import { BaseService, DbService } from '../../lib/service/index';
+
+// import collections
+import { ProductModel, ProductDetail } from '../../model/index';
+
+// import messages
+import cms from '../../cms/product/index';
+
+class Service extends BaseService {
+  registerProduct = async (data) => {
+    try {
+      // const data = { ...req.body };
+
+      const requiredFields = ["name", "supplierId","unitPrice"];
+      this.validateRequired(data, requiredFields);
+
+      const isExist = await DbService.count(ProductModel, { name: data.name, supplierId: data.supplierId });
+      if (isExist) {
+        return this.error(cms.alreadyRegistered);
+      }
+
+
+      const Product = await DbService.create(ProductModel, {
+        name: data.name,
```

Comment:
add checks like data && data.name

Reply:
DbService is correct notation. It doesn't need to be changed